src/sidebar/components/test/FilterSelect-test.js

```diff
@@ -88,4 +92,27 @@ describe('FilterSelect', () => {
     assert.isFalse(wrapper.find('MenuItem').at(1).props().isSelected);
     assert.isTrue(wrapper.find('MenuItem').at(2).props().isSelected);
   });
+
+  it(
+    'should pass a11y checks',
+    checkAccessibility([
+      {
+        content: () => {
+          $imports.$restore();
```

Comment:
This line has the effect of disable `mock-imported-components` making the a11y more realistic.